src/conrad_relaycard/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .card import RelayCard as RelayCard

```

Comment:
the `as ..` is not needed.

Reply:
unfortunately ruff give this warning https://docs.astral.sh/ruff/rules/unused-import/

Is disable this at this lines good for you?

```python
from .card import RelayCard  # noqa: F401
from .exceptions import RelayCardError  # noqa: F401
from .state import RelayState  # noqa: F401
```

Reply:
Yes thats better. Thanks!

Reply:
Done